schema/config-schema.json

```diff
@@ -135,10 +135,17 @@
                 },
                 "capabilities": {
                     "id": "https://opencontainers.org/schema/bundle/process/linux/capabilities",
-                    "type": "array",
-                    "items": {
-                        "$ref": "defs-linux.json#/definitions/Capability"
-                    }
+                    "type": "object",
+					"properties": {
+						"bounding": {
+							"id": "https://opencontainers.org/schema/bundle/process/linux/capabilities/bounding",
+							"$ref": "defs-linux.json#/definitions/Capability"
```

Comment:
This (and `ambient`) should still be arrays.  So:

    "type": "array",
    "items": {
      "$ref": "defs-linux.json#/definitions/Capability"
    }

instead of the line I'm commenting on.